### PR TITLE
Some improvements to the api.

### PR DESCRIPTION
**Pull request by wolfcomp**
- Added in so arguments can be converted to ids instead of using names.
- Added `isMod` field for easier checks for bot makers to make mod only commands
- Added pagination
- Added `/apinext` for current changes these should be out for 2 weeks minimum before loaded over to `/api` to give people time to change to new queries

Places with pagination:
```
followers
channels
channel{
  chatMessages
  bans
  moderationLogs
  moderators
}
```

All pagination except chatMessages has a limit of 1000 currently set where as chatMessages only has 100 as a max

### CODE REVIEW

**Comment by wolfcomp** [comment]
Will fix tests after alpha is live

File: lib/glimesh/graphql/schema.ex [hunk 1]
```diff
@@ -4,6 +4,7 @@ defmodule Glimesh.Schema do
   """
 
   use Absinthe.Schema
+  use Absinthe.Relay.Schema, :modern
```
Comment (clone1018) [inline lib/glimesh/graphql/schema.ex:7]: Is this needed? I think we may only need it on the new lib/glimesh/api/schema.ex

**Comment by clone1018** [comment]
Okay, I went through and removed the files with only line ending changes and I also refactored the functions that return queries to be prefixed with `query_` and added back their `list_` or `get_` counterparts. These changes were done to ensure our queries are happening inside the contexts instead of inside controllers. 

I think this is ready for a final pass of review! Super excited to get this new, much more performant, and scalable API in place. Thanks again for your hard work @wolfcomp !

**Comment by clone1018** [comment]
One more thought, is the inserted_at an atomic enough field to be sorting by for the pagination? I suspect it's possible to have two timestamps be the exact same and you could miss records in the pagination.

File: lib/glimesh/graphql/schema.ex (hunk 1 above)
Comment (wolfcomp) [reply lib/glimesh/graphql/schema.ex:7]: Might have been something I forgot to remove once I moved it to its own schema for relay and old

**Review by clone1018 — APPROVED** [review]

**Pull request merged**